www/profile/inner.js

```diff
@@ -326,10 +353,13 @@ define([
             $mute.html('');
             var isMuted = muted[data.curvePublic];
             if (isMuted) {
-                var unmuteButton = h('button.btn.btn-secondary.cp-app-profile-friend-request', [
-                    h('i.fa.fa-bell'),
-                    Messages.contacts_unmute || 'unmute'
-                ]);
+                var unmuteButton = h('button', 
+                    { class: 'btn btn-secondary cp-app-profile-friend-request', 'aria-labelledby': 'cp-unmute-button' }, 
+                    [
+                        h('i', { class: 'fa fa-bell', 'aria-hidden': 'true' }),
+                        h('span', { id: 'cp-unmute-button' }, Messages.contacts_unmute || 'Unmute')
```

Comment:
In multiple places you added similar ids. I don't think they will create issue here but be careful with generic names, even when you use the `cp-` prefix.
"cp-cancel-button", "cp-unmute-button", etc. are names that could be used in the rest of the product and we could accidentally apply styles here when modifying something else. Usually we try to add the application in the name when describing a very specific element: "cp-profile-cancel-button"